<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>executive_meeting</class>
 <widget class="QDialog" name="executive_meeting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="executive_title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Executive Board Meeting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>110</y>
     <width>191</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="present_label">
        <property name="text">
         <string>Present Members:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QCheckBox" name="name1_label">
        <property name="text">
         <string>Name1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QCheckBox" name="name2_label">
        <property name="text">
         <string>Name2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QCheckBox" name="name3_label">
        <property name="text">
         <string>Name3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QCheckBox" name="name4_label">
        <property name="text">
         <string>Name4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>158</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="point_sheet_label">
      <property name="text">
       <string>Point Sheet:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QPushButton" name="point_sheet_upload">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QCheckBox" name="name5_label">
        <property name="text">
         <string>Name5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QCheckBox" name="name6_label">
        <property name="text">
         <string>Name6</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QCheckBox" name="name7_label">
        <property name="text">
         <string>Name7</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QCheckBox" name="name8_label">
        <property name="text">
         <string>Name8</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QCheckBox" name="name9_label">
        <property name="text">
         <string>Name9</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_14">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="other_label">
      <property name="text">
       <string>Others Present:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <item>
       <widget class="QPushButton" name="other_upload">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="backLabel">
   <property name="geometry">
    <rect>
     <x>-560</x>
     <y>-380</y>
     <width>2031</width>
     <height>1261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backLabel</zorder>
  <zorder>executive_title</zorder>
  <zorder>next_button</zorder>
  <zorder>verticalLayoutWidget_3</zorder>
  <zorder>verticalLayoutWidget_5</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>verticalLayoutWidget_14</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
